exercises/diffie-hellman/canonical-data.json

```diff
@@ -0,0 +1,78 @@
+{ "exercise": "diffie-hellman"
+, "version" : "1.0.0"
+, "comments":
+    [ " In the following descriptions, p and g represent two          "
+    , " prime numbers.                                                "
+    , "                                                               "
+    , " Unless specified otherwise, tests are setup with              "
+    , " new DiffieHellman object using p = 23; g = 5.                 "
```

Comment:
I do not think this sentence is necessary since all the cases in this file seem to be providing `p` and `g` whenever they are necessary... or if they do not, I think they should! If we leave it implicit, it may become unclear which values are needed for which tests. For example, `g` is not necessary in the test of generating a private key.